README.md

```diff
@@ -112,6 +112,8 @@ and flags are:
       (default Info)
   -logprefix string
     	Prefix to log lines before logged messages (default "> ")
+  -maxpayloadsizekb int
+        MaxPayloadSizeKb is the maximum size of payload to be generated by the EchoHandler size= argument (In kilobytes defaults to 256)
```

Comment:
please manually wrap to 80 cols (or I can do it later too it's ok)

Reply:
done.